README.md

```diff
@@ -56,9 +80,9 @@ Give a ⭐️ if you like this project!
 
 ## Acknowledgments
 
-- 
-- 
-- 
+-
+-
+-
 
 ## 📝 License
```

Comment:
- Your link to the license is broken please make sure that you add the li\cense in your project and use the correct link in your readme file it will fix the bag you have  :+1: